### PR TITLE
Sentry

### PR DESCRIPTION
**Pull request by Yatekii**
I added OPTIONAL logging to sentry and would welcome opinions on it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,11 @@ Config file precedence:
 Instead of a TOML file, you can also use a JSON or YAML file. Choose what suits you best!
 
 You can find all available options in the [default.toml](src/config/default.toml). Commented out options are the ones that are `None` by default.
+
+# Sentry logging
+
+We use Sentry to record crash data. This helps us trace crashes better.
+No data will ever be transmitted without your consent!
+All data is transmitted completely anonymous.
```
Comment (dbrgn) [inline README.md:52]:
```suggestion
All data is transmitted completely anonymously.
```

File: README.md [hunk 2]
```diff
@@ -44,3 +44,11 @@ Config file precedence:
...
+This is an OPT-IN feature. On crash, cargo-embed will ask you whether to transmit the data or not. You can also set whether to do this for all times with an environment variable to ommit the message in the future.
```
Comment (dbrgn) [inline README.md:53]:
```suggestion
This is an OPT-IN feature. On crash, cargo-embed will ask you whether to transmit the data or not. You can also set whether to do this for all times with an environment variable to omit the message in the future.
```

**Review by dbrgn — COMMENTED** [review]
I like it, as long as it's strictly opt-in :slightly_smiling_face: 

Where's that sentry server hosted, and who has access to it? Are the crash reports public?

**Comment by Yatekii** [comment]
Thanks for the proofread (do we write this as one word? oO)!
Yes it will be strictly opt in.

We have an OSS sponsored organization on sentry.io. Sadly it cannot be viewed without an account but anyone can join. So if you would like to see what's going on, I'll add you, no questions asked :)

**Review by Tiwalun — APPROVED** [review]
bors r+

**Pull request merged**